code/modules/clothing/glasses/_glasses.dm

```diff
@@ -573,3 +573,38 @@
 	desc = "Lookin' cool."
 	icon_state = "phantom_glasses"
 	inhand_icon_state = "phantom_glasses"
+
+/obj/item/clothing/glasses/dealmaker
+	name = "dealmakers"
+	desc = "Why be the little sponge who hates its $4.99 life when you can be a BIG SHOT?"
+	icon_state = "dealmaker"
+	inhand_icon_state = "dealmaker"
+
+/obj/item/clothing/glasses/dealmaker/equipped(mob/M, slot)
+	. = ..()
+	if (slot == ITEM_SLOT_EYES)
+		RegisterSignal(M, COMSIG_MOB_SAY, .proc/handle_speech)
+	else
+		UnregisterSignal(M, COMSIG_MOB_SAY)
+
+/obj/item/clothing/glasses/dealmaker/dropped(mob/M)
+	. = ..()
+	UnregisterSignal(M, COMSIG_MOB_SAY)
+
+/obj/item/clothing/glasses/dealmaker/proc/handle_speech(datum/source, list/speech_args) //this should probably be changed at some point via some sort of refactor so that all clothing can modify speech
+	SIGNAL_HANDLER
+	var/message = speech_args[SPEECH_MESSAGE]
+	if(message[1] != "*")
+		message = " [message]"
+		var/list/salesman_words = strings("salesman_replacement.json", "salesman")
```

Comment:
this should be a define, we store them in _DEFINES/misc.dm. really sucks that these are in a misc.dm

![image](https://user-images.githubusercontent.com/40974010/135397658-6aabb886-c11b-4cc5-94d6-2733990e7eb6.png)